network/monuments.csv

```diff
@@ -1,5 +1,5 @@
 Mark,Domes Number,Mark Type,Type,Ground Relationship,Foundation Type,Foundation Depth,Start Date,End Date,Bedrock,Geology
-2004,,Forced Centering,Steel Mast,0,Reinforced Concrete,0,2000-06-19T00:00:00Z,9999-01-01T00:00:00Z,,
+2004,,Forced Centering,Steel Mast,0,Reinforced Concrete,0,2000-06-19T00:00:00Z,2024-07-05T00:00:00Z,,
```

Comment:
has the monument been removed from the site? If not, the end date here can remain "open"

Reply:
As far as I know the monument has been removed (from one of the comments in the ticket: `2004 mark is still within the fenced area if it ever needs to be utilised, but the monument has been returned to GNS`)
The thing I am concerned is if we should close site in the `marks.csv` table or not. The geodetic mark is still there, but I don't think we will occupy it again. What do you think @elidana ?

Reply:
oh no, if there was some form of antenna mount installed over the geodetic mark on the ground then yes, agree with you that the "monumnet' can be considered as closed as well! Sorry for the confusion!

Reply:
No worries! This is exactly what I wanted to discuss. Will push the PR to the review.